book/chapter-00.md

```diff
@@ -26,30 +26,30 @@ We can use these tools to push forth a variety of good and bad outcomes:
+That’s cool… but how do we govern and design decentralized networks? With these open networks, we want to be able to open participation to the public internet and incentivize participation in the security and maintenance of the network (running nodes, mining, etc).
 
-In a centralized payment processor, we trust the central authority  to verify transactions and update balances. However, in a decentralized payment processor, everyone runs everything by listening for and downloading new transactions. But in what order? Lack of agreement on ordering leads to a potential problem which prevents everyone from agreeing on a common state of the network.
+In a centralized payment processor, we trust the central authority to verify transactions and update balances. However, in a decentralized payment processor, everyone runs everything by listening for and downloading new transactions. But in what order? Lack of agreement on the order of transactions leads to a potential problem - the inability of everyone to agree on a common state of the network.
 
   
 |               | Order 1                                                             | Order 2                                                              |
 |---------------|---------------------------------------------------------------------|----------------------------------------------------------------------|
 | Initial State | Aparna has 10 eth, Aparna sends Jing 8 eth, Aparna sends Karl 8 eth | Aparna has 10 eth, Aparna sends Karl 8 eth, Aparna sends Jing 8 eth, |
 | Final State   | Aparna has 2 eth, Jing has 8 eth, Karl has 0 eth                    | Aparna has 2 eth, Karl has 8 eth, Jing has 0 eth         |
 
-Two drastically different final states arise from a lack of agreement on ordering
+Note that two drastically different final states arise from a lack of agreement on transaction ordering.
 
-It’s not enough to have decentralized verification - we also need decentralized consensus on the verification of the order that the code gets executed. In this course we will start with a centralized implementation of a simple payment system. We’ll call it Web 2.0 PayPal. As the course progresses, we will turn it into a decentralized payment protocol - and call it Web 3.0 PayPal because buzzwords are easy signals for naming. But before we embark on that journey, let’s take a high level whirl through cryptoeconomics in bitcoin. 
+It’s not enough to have decentralized verification - we also need decentralized consensus on the verification of the order that the code gets executed in. In this course, we will start with a centralized implementation of a simple payment system. We’ll call it Web 2.0 PayPal. As the course progresses, we will turn it into a decentralized payment protocol and call it Web 3.0 PayPal (because buzzwords are easy signals for naming). But before we embark on that journey, let’s take a high level whirl through cryptoeconomics in Bitcoin. 
```

Comment:
re - capitalization of "Bitcoin"

My understanding is that the protocol is "Bitcoin," and the token is "bitcoin."